### PR TITLE
fix script error on adding new element to non-added to DOM 'body' element (close #296)

### LINKED ISSUE #296: Script error on adding new element to non-added to DOM 'body' element
Script to reproduce:
```javascript
var iframe = document.createElement('iframe');

    iframe.id = 'test_unique_id_l7ajkl';
    iframe.addEventListener('load', function () {
        var iframeDocument = this.contentDocument;

        iframeDocument.documentElement.removeChild(iframeDocument.body);

        var newIframeBody = iframeDocument.createElement('body');
        var div1          = iframeDocument.createElement('div');
        var div2          = iframeDocument.createElement('div');

        newIframeBody.appendChild(div1);
        newIframeBody.insertBefore(div2, null);

        iframe.parentNode.removeChild(iframe);
```

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @inikulin @churkin @VasilyStrelyaev

### CODE REVIEW

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/node/dom-processor-test.js [hunk 1]
```diff
@@ -432,3 +432,27 @@ test('Node.replaceChild must be overridden (GH-264)', function () {
     style.replaceChild(styleTextNode2, styleTextNode1);
     ok(style.innerHTML.indexOf(urlUtils.getProxyUrl('/image2.png')) > -1);
 });
+
+asyncTest('Script error on adding new element to non-added to DOM "body" element (GH-296)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]:
```
asyncTest('Script error when a new element is added to a "body" element that is not in the DOM (GH-296)', function () {
```

Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: Or should we rather begin test names with `No error occurs when...` if this is what we expect?

Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/dom-processor-test.js]: Or just `Adding a new element to a "body" element that is not in the DOM`

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Comment by churkin** [comment]
\r

**Comment by miherlosev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by miherlosev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Pull request merged**